# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1274333265 -7200
#      Thu May 20 07:27:45 2010 +0200
# Node ID 45b72e631cb52231a73a6a12583b3f50360b7587
# Parent  9852264314d1d98bb3872598b2ae7e9ad0b493f4
avoid unused param warnings in onCleanup

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,15 @@
+2010-05-20  Jaroslav Hajek  <highegg@gmail.com>
+
+	* DLD-FUNCTIONS/onCleanup.cc (octave_oncleanup::load_ascii,
+	octave_oncleanup::load_binary, octave_oncleanup::load_hdf5,
+	octave_oncleanup::save_ascii, octave_oncleanup::save_binary,
+	octave_oncleanup::save_hdf5): Comment out unused params.
+
 2010-05-18  Jaroslav Hajek  <highegg@gmail.com>
 
 	* defun-int.h (octave_auto_shlib): Make members public.
 	* DLD-FUNCTIONS/onCleanup.cc: New source.
 	* DLD-FUNCTIONS/module-files: Add it here.
 
 2010-05-18  Jaroslav Hajek  <highegg@gmail.com>
 
diff --git a/src/DLD-FUNCTIONS/onCleanup.cc b/src/DLD-FUNCTIONS/onCleanup.cc
--- a/src/DLD-FUNCTIONS/onCleanup.cc
+++ b/src/DLD-FUNCTIONS/onCleanup.cc
@@ -179,55 +179,56 @@ octave_oncleanup::map_value (void) const
 
 static void 
 warn_save_load (void)
 {
   warning ("onCleanup: load and save not supported");
 }
 
 bool 
-octave_oncleanup::save_ascii (std::ostream& os)
+octave_oncleanup::save_ascii (std::ostream& /* os */)
 {
   warn_save_load ();
   return true;
 }
 
 bool 
-octave_oncleanup::load_ascii (std::istream& is)
+octave_oncleanup::load_ascii (std::istream& /* is */)
 {
   warn_save_load ();
   return true;
 }
 
 bool 
-octave_oncleanup::save_binary (std::ostream& os, bool& save_as_floats)
+octave_oncleanup::save_binary (std::ostream& /* os */, bool& /* save_as_floats */)
 {
   warn_save_load ();
   return true;
 }
 
 
 bool 
-octave_oncleanup::load_binary (std::istream& is, bool swap, 
-                               oct_mach_info::float_format fmt)
+octave_oncleanup::load_binary (std::istream& /* is */, bool /* swap */, 
+                               oct_mach_info::float_format /* fmt */)
 {
   warn_save_load ();
   return true;
 }
 
 #if defined (HAVE_HDF5)
 bool 
-octave_oncleanup::save_hdf5 (hid_t loc_id, const char *name, bool save_as_floats)
+octave_oncleanup::save_hdf5 (hid_t /* loc_id */, const char * /* name */, 
+                             bool /* save_as_floats */)
 {
   warn_save_load ();
   return true;
 }
 
 bool 
-octave_oncleanup::load_hdf5 (hid_t loc_id, const char *name)
+octave_oncleanup::load_hdf5 (hid_t /* loc_id */, const char * /* name */)
 {
   warn_save_load ();
   return true;
 }
 #endif
 
 void 
 octave_oncleanup::print (std::ostream& os, bool pr_as_read_syntax) const
